### DISCUSSION TITLE
Asking for Synology DSM

### DISCUSSION THREAD

**Discussion by naruko-hstk** [marcopeocchi/yt-dlp-web-ui#211] [Q&A] [answered]
I want to run this on my DSM
How to do that?

**Comment by marcopiovanello** [comment, marked as answer]
Hello @naruko-hstk,

There's a guide on Synology's site to how to run docker apps (https://kb.synology.com/ja-jp/DSM/help/Docker/docker_container?version=6)

The label to search in the registry section is `marcobaobao/yt-dlp-webui`

**Comment by marcopiovanello** [comment]
Hello @naruko-hstk, any update on this?

**Reply by naruko-hstk** [reply]
Tested.
It can run.
Thanks.
